pulsar-flink-connector/src/main/java/org/apache/flink/table/catalog/pulsar/factories/PulsarCatalogFactory.java

```diff
@@ -66,6 +66,8 @@ public Catalog createCatalog(String name, Map<String, String> properties) {
         props.add(CATALOG_ADMIN_URL);
         props.add(CATALOG_STARTUP_MODE);
         props.add(CATALOG_DEFAULT_PARTITIONS);
+        props.add(org.apache.flink.streaming.connectors.pulsar.table.PulsarOptions.SCAN_STARTUP_MODE.key());
+        props.add(FormatDescriptorValidator.FORMAT);
```

Comment:
Do we need add support for those configurations of key and value like key.xxx

Reply:
Yes, support needs to be added.